biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/BiopetQScript.scala

```diff
@@ -121,6 +122,7 @@ trait BiopetQScript extends Configurable with GatkLogging { qscript: QScript =>
       if (!i.file.isAbsolute) Logging.addError(s"Input file should be an absolute path: ${i.file}")
     }
 
+    logger.info("Set stdout file when not set")
```

Comment:
I assume you want to say that `stdout` is automatically set because it is not explicitly set by the user?